<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
    <title>SubNucPred:: A web-server for prediction of sub-nuclear localization of proteins:: Help</title>
    <link href="style.css" rel="stylesheet" type="text/css" />
  </head>
  <body>
    <div id="topPan">
      <a href="index.html"><img src="images/logo.jpg" alt="SubNucPred" width="245" height="37" border="0"  class="logo" title="SubNucPred"/></a>
      <p>A webserver for prediction of sub-nuclear locations</p>    
      
      <div id="topContactPan">
      </div>
      <div id="topMenuPan">
	<div id="topMenuLeftPan"></div>
	<div id="topMenuMiddlePan">
	  <ul>
	    <li><a href="index.html">Home</a></li>
	    <li><a href="submission.html">Submission</a></li>
	    <!--<li class="home">Help</li>-->
	    <li><a href="help.html">Help</a></li>
	    <li><a href="download.html">Standalone</a></li>
	    <li><a href="download.html">Dataset</a></li>
	    <li><a href="algo.html">Algorithm</a></li>
	    <li><a href="developers.html">Developers</a></li>
	    <li><a href="contact.html">Contact</a></li>
	  </ul>
	</div>
	  <div id="topMenuRightPan"></div>
      </div>
    </div>
    <div id="bodyPan">
      <div id="bodyCENTERPan">
  	<h2><span>How to Use</span></h2><br>
	  <p><b>Query Data for Prediction</b><br>SubNucPred only requires protein sequence in FASTA format (upto maximum 25 sequences at a time). If user submit more than 25 sequences, prediction is done for first 25 sequences. Sequences can be provided either by pasting to the designated text box or can be uploaded as a text file. Following (Figure 1) is an example showing different options of data submission.</p><br>
	      <p><b>SVM Threshold</b><br>If user want prediction with high sensitivity, a low threshold should be selected. At low threshold, more prediction will be reported, but rate of false positive prediction will also be high. If user want only those predictions which are highly specific, high threshold value should be selected. At high threshold although the number of predictions will be low, but rate of false positive prediction will also be low.</p><br>
		  <p><b>Example Sequence</b></p><p>>sp|P07199|CENPB_HUMAN Major centromere autoantigen B OS=Homo sapiens GN=CENPB PE=1 SV=2</br>
MGPKRRQLTFREKSRIIQEVEENPDLRKGEIARRFNIPPSTLSTILKNKRAILASERKYG</br>
VASTCRKTNKLSPYDKLEGLLIAWFQQIRAAGLPVKGIILKEKALRIAEELGMDDFTASN</br>
GWLDRFRRRHGVVSCSGVARARARNAAPRTPAAPASPAAVPSEGSGGSTTGWRAREEQPP</br>
SVAEGYASQDVFSATETSLWYDFLPDQAAGLCGGDGRPRQATQRLSVLLCANADGSEKLP</br>
PLVAGKSAKPRAGQAGLPCDYTANSKGGVTTQALAKYLKALDTRMAAESRRVLLLAGRLA</br>
AQSLDTSGLRHVQLAFFPPGTVHPLERGVVQQVKGHYRQAMLLKAMAALEGQDPSGLQLG</br>
LTEALHFVAAAWQAVEPSDIAACFREAGFGGGPNATITTSLKSEGEEEEEEEEEEEEEEG</br>
EGEEEEEEGEEEEEEGGEGEELGEEEEVEEEGDVDSDEEEEEDEESSSEGLEAEDWAQGV</br>
VEAGGSFGAYGAQEEAQCPTLHFLEGGEDSDSDSEEEDDEEEDDEDEDDDDDEEDGDEVP</br>
VPSFGEAMAYFAMVKRYLTSFPIDDRVQSHILHLEHDLVHVTRKNHARQAGVRGLGHQS</br>
>sp|Q15172|2A5A_HUMAN Serine/threonine-protein phosphatase 2A 56 kDa regulatory subunit alpha isoform OS=Homo sapiens GN=PPP2R5A PE=1 SV=1</br>
MSSSSPPAGAASAAISASEKVDGFTRKSVRKAQRQKRSQGSSQFRSQGSQAELHPLPQLK</br>
DATSNEQQELFCQKLQQCCILFDFMDSVSDLKSKEIKRATLNELVEYVSTNRGVIVESAY</br>
SDIVKMISANIFRTLPPSDNPDFDPEEDEPTLEASWPHIQLVYEFFLRFLESPDFQPSIA</br>
KRYIDQKFVQQLLELFDSEDPRERDFLKTVLHRIYGKFLGLRAFIRKQINNIFLRFIYET</br>
EHFNGVAELLEILGSIINGFALPLKAEHKQFLMKVLIPMHTAKGLALFHAQLAYCVVQFL</br>
EKDTTLTEPVIRGLLKFWPKTCSQKEVMFLGEIEEILDVIEPTQFKKIEEPLFKQISKCV</br>
SSSHFQVAERALYFWNNEYILSLIEENIDKILPIMFASLYKISKEHWNPTIVALVYNVLK</br>
TLMEMNGKLFDDLTSSYKAERQREKKKELEREELWKKLEELKLKKALEKQNSAYNMHSIL</br>
SNTSAE</br>
>sp|O74791|GRN1_SCHPO GTPase grn1 OS=Schizosaccharomyces pombe (strain 972 / ATCC 24843) GN=grn1 PE=1 SV=1</br>
MVSLKKKSKRRTTRLRSRIEKKAAESKRKQKRADKKNPQWKSRIPKDPGIPNSFPYKDKI</br>
LAEIEEQKRIREEEKLARRASGQVDAAMEEEDAVDENGSLMISKIAEAAQASNPDDEEEF</br>
VMEEDNLGEAPLLVDSESYEASVKADTSRKAYDKEFKKVVEASDVILYVLDARDPEGTRS</br>
KDVERQVLASSAEEKRLIFVINKIDLVPSEVLNKWVTYLRNFFPTIPMRSASGSGNSNLK</br>
HQSASASSTISNLLKSLKSYSAKKKLKSSLTVGVIGYPNVGKSSVINALVNRSANGRSAP</br>
CPAGNVAGMTTSLREVKLDNKLRLVDSPGIVFPSSDSKDDLYRLVMLNAVSSTKVDDPVA</br>
VASYILQFLSRVPGQLERMFQRYELPPLLNTSDIDTATDFLVNIARKRGRLGRGGIPNLN</br>
AAANIVINDWHAGRIEWWAEPEVINEKNSSEVQDTQIVTEWAKEFDLNDF</br>
</p><br>
		  </div>
		  <div id="bodyLOWERPan">
		<table align='center' width='100%'>
		  <tr border='2'><td><a target="_blank"><img align="center" border="1" src="images/subpage.png" width="700" height="410"></td></a></tr></table>
		  <br><b>Figure 1.</b> SubNucPred submission page showing how to insert input sequences.<br><br>
		      </div>
<div id="bodyCENTERPan">
  <p><b>Prediction Result</b><br>In response to the submitted protein sequence(s), SubNucPred generates the prediction result in tabular form. In case prediction is done on the basis of unique Pfam domain, the result shows predicted location and domain based prediction in a table. If the prediction is done on the basis of SVM score, the result shows predicted location and their corresponding SVM score. Here is a sample of prediction done in response to three query proteins (Figure 2).</p>
</div><br>
  <div id="bodyLOWERPan">
    <table align='center' width='100%'>
      <tr border='2'><td><a target="_blank"><img align="center" border="1" src="images/result.png" width="700" height="410"></td></a></tr></table>
      <br><b>Figure 2.</b> Screenshot of SubNucPred prediction result.<br><br>
	  </div>
	</body>
      </html>
		  
